src/pages/contact.svelte

```diff
@@ -11,7 +11,7 @@
             <div class="md:px-4 py-4 space-y-4">
                 <div>
                     <label for="{ emailName }" class="block text-left text-gray-700">{ $_('pages.contact.email') }</label>
-                    <input type="text" name="{ emailName }" id="{ emailName }"
+                    <input type="text" name="{ emailName }" id="{ emailName }" placeholder="{ $_('pages.contact.email') }"
```

Comment:
```suggestion
                    <input type="text" name="{ emailName }" id="{ emailName }"
```

I think I will remove here a placeholder - input label clearly says what type of field it is. What's more, for UX is better to have less placeholders - without them form is more transparent.
EDIT:
If we really want placeholder, we should change it to something more fancy, because repeating label text in placeholder is meaningless.  Or we can redesign placeholders to something like this:

![image](https://user-images.githubusercontent.com/56546832/229707736-e519f0cf-02ce-4711-9429-fd312f2decc9.png)